<launch>
    <arg name="veh" doc="Name of vehicle. ex: megaman"/>
    <arg name="output" default="screen"/>
    <param name="$(arg veh)/board_name" value="ominibot_car"/>
    <node ns="$(arg veh)" pkg="ominibot_car" type="wheels_ominibot_driver_node.py" name="wheels_driver_node" output="$(arg output)" clear_params="true" required="true"/>
    
    <!-- Subscriptions -->
    <!-- "~wheels_cmd": rosky_msgs/WheelsCmd. Command for the wheels -->

</launch>
